artemis-server/src/test/java/org/apache/activemq/artemis/core/persistence/impl/journal/OperationContextUnitTest.java

```diff
@@ -422,4 +422,43 @@ public void onError(final int errorCode, final String errorMessage) {
       Assert.assertEquals(0, operations.get());
    }
 
+   @Test
+   public void testContextWithoutStoreOperationTrackers() {
+      ExecutorService executor = Executors.newSingleThreadExecutor(ActiveMQThreadFactory.defaultThreadFactory(getClass().getName()));
+
+      OperationContextImpl.resetMaxStoreOperationTrackers();
```

Comment:
Slight nitpick but I think it would be better if it didnt do this reset first, and so verified the default behaviour was as expected?